### PR TITLE
feat: add no-verify in push tag workflow

### LINKED ISSUE #112: Add `--no-verify` in push tag workflow

### PR DESCRIPTION
**Pull request by spaenleh**
This PR runs the push tag with the `--no-verify` flag to skip hooks.

closes #112

### CODE REVIEW

**Review by dialexo — COMMENTED** [review]
This workflow is called in each repo specifically if I'm not mistaken? However does that mean that we make the assumption that calling `yarn release:<type>` is available in all repositories? Does adding this `no-verify` flag here also make the assumption that all repos use the same release tool? Is that desirable?

**Comment by spaenleh** [comment]
@dialexo yes the assumption is that all repos that call this workflow provide the necessary scripts to create releases using `standard-version`. However, I think we should move towards using release-please even for frontends (currently all libraries are using it to manage their versioning). I have experimented to use it in the `graasp-app-code-capsule` and it integrated into our current stack based system very well with the added benefit of creating more legible Releases in GitHub. Also since the versions are managent by automatic PRs it is a bit more controlled when and how you create a new version and tag.

In conclusion, yes a lot of assumptions are made on the available tools. This PR was more to mitigate problems when creating tags (with husky mainly). But i think it opened an interesting discussion.

**Review by dialexo — APPROVED** [review]
I'll approve for now under the assumption that the flag will either fail workflows that do not use standard release (which is fine), or that the flag should not affect its behaviour (which is also fine). Hopefully we won't run into a scenario where the flag has another meaning with a different tool in the future. Though I think we should make sure to carefully keep each system within its boundaries.
